examples/structured_data/classification_with_tfdf.py

```diff
@@ -146,23 +160,14 @@
 """
 
 
-def prepare_sample(features, target, weight):
-    for feature_name in features:
-        if feature_name in CATEGORICAL_FEATURES_WITH_VOCABULARY:
-            if features[feature_name].dtype != tf.dtypes.string:
-                # Convert categorical feature values to string.
-                features[feature_name] = tf.strings.as_string(features[feature_name])
-    return features, target, weight
-
-
 def run_experiment(model, train_data, test_data, num_epochs=1, batch_size=None):
 
     train_dataset = tfdf.keras.pd_dataframe_to_tf_dataset(
         train_data, label=TARGET_COLUMN_NAME, weight=WEIGHT_COLUMN_NAME
-    ).map(prepare_sample, num_parallel_calls=tf.data.AUTOTUNE)
+    )
     test_dataset = tfdf.keras.pd_dataframe_to_tf_dataset(
         test_data, label=TARGET_COLUMN_NAME, weight=WEIGHT_COLUMN_NAME
-    ).map(prepare_sample, num_parallel_calls=tf.data.AUTOTUNE)
+    )
 
     model.fit(train_dataset, epochs=num_epochs, batch_size=batch_size)
```

Comment:
In this example, I would do the division in double (float64) and then cast the result to float32.

Reply:
Will do

Reply:
Done